rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,435 @@
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
+
+The survey MUST be announced to the membership of the Human Cell Atlas (through its Registration mailing list) with a minimum of a two week last call for responses.
+
+At the conclusion, UX MUST summarize insights from the survey and present at the next DCP PM meeting. Following a one week review to incorporate feedback, the survey MUST be shared by UX with the membership of the Human Cell Atlas and announced on the *HumanCellAtlas #dcp* slack channel.
+
+UX MAY also share critical UX Insights from other sources independent of the Survey for consideration by the Project Leads during the creation of the Roadmap.
+
+#### Roadmap Retrospective
+
+The Roadmap Retrospective occurs during the recurring Project Leads meeting and not in a special session. The Roadmap Shepherd MUST add the Retrospective to the agenda and faciliate.
+
+In preparation, the Project Leads MUST independently review the prior year’s Product Roadmap, Release Retrospective summaries, and appropriate supporting documentation and then share their significant findings and learnings with their peers during the Retrospective.
+
+Findings may include: Themes that were incomplete and the accompanying reason, incorrectly sized work in the Engineering Plan, etc. These findings SHOULD be utilized as a reference class for the next Product Roadmap.
+
+Project Leads MUST identify a Roadmap Shepherd during the Retrospective. The Shepherd MUST NOT be a Project Lead, to ensure the neutrality of the content for the Roadmap.
+
+The Shepherd is responsible for:
+* Managing the timeline for the creation of the draft Roadmap
+* Facilitating all meetings related to the creation of the draft Roadmap 
+* Documenting important decisions, discussions, and questions 
+* Assigning action items to move toward the creation of the Roadmap, should no Project Lead volunteer, and following up with them through completion
+
+### Drafting the Roadmap
+[Drafting the Roadmap]:#drafting-the-roadmap
+
+Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective.
+
+To create the draft Product Roadmap, the Roadmap Shepherd:
+- Forks the *HumanCellAtlas dcp-community* repository
+- Copies `rfcs/roadmap-template.md` to `rfcs/text/0000-roadmap-YYYY.md` - where "YYYY" is the target year for the roadmap. 
+
+The Roadmap Shepherd MUST collate all themes into the draft which is an Informational RFC. 
+
+Project Leads MUST review the Themes as a collective and distill the revised list to a lesser number of themes. This is achieved through a consensus building process, which is implemented by the Roadmap Shepherd and occurs during specifically noted Project Lead meetings, outlined below:
+
+* Explore the Themes, search for commonalities, and document concerns and issues 
```

Comment:
I think that narrowing down the list of themes should involve having a rough understanding of impact on each component so any one component is not overloaded.
Also, "lesser number of themes" is a little vague. I think there is a better way to describe how many themes are ideal - you want enough so that everyone feels they know their priorities without anyone feeling overloaded

Reply:
For overloaded components, I think that is a concern. There are at least two ways to address it. (1) Narrowing down themes (if some work is not essential), another is cross-team support (if all work is essential) but either way "team capacity" would be a nice thing to include here.

I think we can get a feel for team velocity at the roadmap level as we get a little more of a history of execution. I would encourage us to use that as one feature of commenting on team capacity. I do not think this needs to be included however as it may be too prescriptive.